noir-projects/noir-protocol-circuits/crates/private-kernel-lib/src/private_call_data_validator.nr

```diff
@@ -187,17 +158,11 @@ impl PrivateCallDataValidator {
         let min_revertible_side_effect_counter = public_inputs.min_revertible_side_effect_counter;
         // No need to check that the min_revertible_side_effect_counter falls in the counter range of the private call.
         // It is valid as long as it does not fall in the middle of any nested call.
-        validate_split_private_call_requests(
+        validate_clean_split_ranges(
             min_revertible_side_effect_counter,
             first_revertible_private_call_request_index,
-            self.data.private_call_stack,
-            self.array_lengths.private_call_stack_hashes
-        );
-        validate_split_public_call_requests(
```

Comment:
Removing the check for public call requests as their end counters are irrelevant and their data won't be divided min_revertible_side_effect_counter.